_data/simple-icons.json

```
@@ -11877,6 +11877,16 @@
             "hex": "337AB7",
             "source": "https://www.spoj.com"
         },
+        {
+            "title": "SpigotMC",
+            "hex": "ED8106",
+            "source": "https://www.spigotmc.org",
+            "aliases": {
+                "aka": [
+                    "Spigot"
+                ]
+            }
```

```suggestion
            "source": "https://www.spigotmc.org"
```

I'd remove aliases from this, it'll show up already if someone were to search 'Spigot'.